book/src/introduction.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+Harbor is a system for:
+
+- Collecting, categorizing, and storing SBOMs. The process of collecting SBOMs is called 
+  **_ingestion_**.
+- Analyzing SBOMs to identify associated vulnerability and supply chain data. The process of 
+  identifying and storing vulnerability and supply chain data associated with SBOMs is called 
+  **_enrichment_**.
+
+### Compliance
+
+Harbor was born out of the [Cyber Executive Order](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/)
+which mandates that all federal agencies must procure and catalog SBOMs for all software they 
+purchase or develop. Compliance with the Executive Order is achieved through the **_ingestion_** 
+of SBOMs.
+
+### Operational Security
+
+While **_ingestion_** is a mandatory first step, that activity alone does nothing to secure an
```

Comment:
Storage here too.  We might consider referring to the whole process as `I&S`.  That way the processes are coupled forever.

Reply:
Updated in next push